src/customdata.jl

```diff
@@ -0,0 +1,36 @@
+struct CustomVars <: MOI.AbstractModelAttribute end
+struct CustomConstrs <: MOI.AbstractModelAttribute end
+
+addcustomvars!(model, customvars::Vector{DataType}) = MOI.set(
+    model, CustomVars(), customvars
+)
+
+addcustomconstrs!(model, customconstrs::Vector{DataType}) = MOI.set(
+    model, CustomConstrs(), customconstrs
+)
+
+customvars(model) = MOI.get(model, CustomVars())
+
+customconstrs(model) = MOI.get(model, CustomConstrs())
```

Comment:
Can you add a docstring for these four methods ?

Reply:
Should I keep the names `addcustomvars!`/`addcustomconstrs!` and `customvars`/`customconstrs`? I added `customvars`/`customconstrs`, but for me something like `setcustomvars!`/`setcustomconstrs!` and `getcustomvars`/`getcustomconstrs` would maybe describe better these methods.

Reply:
I think `customvars` and `customvars!` are ok to keep consistency with other method names.